source/server/http/admin.cc

```diff
@@ -575,13 +574,66 @@ std::string AdminImpl::runtimeAsJson(
   return strbuf.GetString();
 }
 
+Http::Code AdminImpl::handlerHystrixEventStream(const std::string&,
```

Comment:
Quick drive by comment. This file is getting huge and will continue to get huger. Can you put all the new Hystrix admin stuff into a dedicated file, potentially in a new admin directory? We will need to start decomping the rest of this file also as we add more stuff to it.

Reply:
moved hystrix stuff from admin.cc - commit a05aea1a9388df75e87183c402dd323ce1c8e14b